recipes/fasttext/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "fasttext" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8c4ba14372d160fe7de902299caac664b7c1af1778601e17ca18ca2081b0e253" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 431c9e2.tar.gz
+  url: https://github.com/facebookresearch/fastText/archive/431c9e2.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc   # [linux]
+    - llvm  # [osx]
```

Comment:
This is from `defaults` and probably isn't what you want. Try just using the system compiler on Mac too (with `toolchain` as a build requirement).